app/javascript/pages/account/account-settings.tsx

```diff
@@ -220,7 +281,14 @@ const DateOfBirthSection = ({ user, setUser }: SectionProps) => {
       DOB: [dob.birthYear, dob.birthMonth, dob.birthDay].join("-"),
     }
 
-    await updateNameOrDOB(newUser, saveProfile, setUser, setLoading)
+    await updateNameOrDOB(
+      newUser,
+      saveProfile,
+      setUser,
+      setLoading,
+      handleBanners,
+      "dobSavedBanner"
```

Comment:
Nice! I think this can be simplified more so that it is just passed as one function. Something like 
```suggestion
     () => handleBanners("dobSavedBanners")
```
and the type on the `updateNameOrDOB` could be: `bannersCallback?: () => void`